src/template/src/fixtures/version.json

```diff
@@ -385,6 +425,37 @@
           {
             "fillType": "gradient",
             "blendMode": "normal",
+            "fill": {
```

Comment:
Same here. We already have this data on the `gradient` field.

Reply:
Surprisingly, sample data and fixtures are different. I copied and pasted and didn't notice the difference like here.